### PR TITLE
Feature/disable ssl check

### PR DESCRIPTION
**Pull request by tdesfont**

### CODE REVIEW

**Comment by tdesfont** [comment]
First draft, need to add some unit tests on the configuration

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
The PR is not working.
The parameter ignore_ssl_check should be used to tell the tableau hyper client or the requests package to ignore SSL Certificates.

**Comment by tdesfont** [comment]
Changed !

File: parameter-sets/tableau-server-connection/parameter-set.json [hunk 1]
```diff
@@ -35,12 +35,22 @@
...
+        },
         {
             "name": "ssl_cert_path",
             "label": "SSL certificate",
             "type": "STRING",
             "description": "(optional) Full path to your tableau SSL certificate",
-            "mandatory": false
+            "mandatory": false,
+            "visibilityCondition": "model.ignore_ssl == false"
+
```
Comment (MayeulRousselet) [inline parameter-sets/tableau-server-connection/parameter-set.json:53]: remove this blank line

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
Just a nitpick request and we're gooood

File: parameter-sets/tableau-server-connection/parameter-set.json (hunk 1 above)
Comment (tdesfont) [reply parameter-sets/tableau-server-connection/parameter-set.json]: Done in 69faa5f

**Pull request merged**